modules/android/tests/test_android_project.lua

```diff
@@ -41,7 +42,9 @@
 	end
 
 	function suite.rttiOn()
+		exceptionhandling "On"
 		rtti "On"
+
```

Comment:
This newline breaks consistency with the other tests

Reply:
Fixed.